cypress/support/fragments/settings/orders/acquisitionMethods.js

```diff
@@ -0,0 +1,21 @@
+import uuid from 'uuid';
+import getRandomPostfix from '../../../utils/stringTools';
+
+export default {
+    defaultAcquisitionMethod: {
+        id: uuid(),
+        value: `AU_name_${getRandomPostfix()}`,
+      },
+    createNewAcquisitionMethodViaAPI: (acquisitionMethod) => cy.okapiRequest({
+        method: 'POST',
+        path: 'orders/acquisition-methods',
+        body: acquisitionMethod,
+        isDefaultSearchParamsRequired: false,
+      }).then(response => response.body),
+  
+    deleteAcquisitionMethodViaAPI: (id) => cy.okapiRequest({
+        method: 'DELETE',
+        path: `orders/acquisition-methods/${id}`,
+        isDefaultSearchParamsRequired: false,
+      }),
+    };
```

Comment:
please could you check expected indentations and format of all 3 files. there are errors that eslint throws